### DISCUSSION TITLE
Erro of init_bulk: WARNING : in directory XXX/POSCAR.02x02x02/02.md/sys-0008-0008-0024 nforce in OUTCAR is not equal to settings in INCAR

### DISCUSSION THREAD

**Discussion by biglinn** [deepmodeling/dpgen#1468] [Q&A]
When I produce initial data for model trainling. I perform the init_bulk, but there is an erro:
_RuntimeError: MD dir: XXX/POSCAR.02x02x02/02.md: find no valid outcar in sys sys-0008-0008-0024, check if your vasp md simulation is correctly done_
And I look for the file 'dpgen.log'. There is warnning information:
_WARNING : in directory XXX/POSCAR.02x02x02/02.md/sys-0008-0008-0024 nforce in OUTCAR is not equal to settings in INCAR_

I wonder why this erro appears. I've followed the tourial to keep 'md_nsteps' and 'NSW' in agreement.

Following is my param.json:

1. {
2.     "stages" : [1,2,3,4],
3.     "elements": ["K","Ta","O"],
4.     "super_cell": [2, 2, 2],
5.     "from_poscar": true,
6.     "from_poscar_path": "./POSCAR",
7.     "potcars": ["./POTCAR_K", "./POTCAR_Ta","./POTCAR_O"],
8.     "relax_incar": "./INCAR_rlx",
9.     "md_incar" : "./INCAR_md",
10.     "skip_relax": true,
11.     "scale": [1.00],
12.     "pert_numb": 100,
13.     "pert_box": 0.03,
14.     "pert_atom": 0.01,
15.     "md_nstep" : 50,
16.     "coll_ndata": 10000,
17.     "type_map" : ["K","Ta","O"],
18.     "_comment": "that's all"
19. }

**Comment by Normadp** [comment]
Hello, has your problem been solved? I've encountered the same issue.
